src/Umbraco.Web.UI.Client/src/common/resources/media.resource.js

```diff
@@ -554,28 +554,88 @@ function mediaResource($q, $http, umbDataFormatter, umbRequestHelper) {
                 'Failed to retrieve media items for search: ' + query);
         },
 
-        /**
-         * @ngdoc method
-         * @name umbraco.resources.mediaResource#getReferences
-         * @methodOf umbraco.resources.mediaResource
-         *
-         * @description
-         * Retrieves references of a given media item.
-         *
-         * @param {Int} id id of media node to retrieve references for
-         * @returns {Promise} resourcePromise object.
-         *
-         */
-        getReferences: function (id) {
+        getPagedContentReferences: function (id, options) {
+
+            var defaults = {
+                pageSize: 25,
+                pageNumber: 1
+            };
+            if (options === undefined) {
+                options = {};
+            }
+            //overwrite the defaults if there are any specified
+            angular.extend(defaults, options);
+            //now copy back to the options we will use
+            options = defaults;
+
+            return umbRequestHelper.resourcePromise(
+                $http.get(
+                    umbRequestHelper.getApiUrl(
+                        "mediaApiBaseUrl",
+                        "GetPagedContentReferences",
+                        {
+                            id: id,
+                            pageNumber: options.pageNumber,
+                            pageSize: options.pageSize
+                        }
+                    )),
+                "Failed to retrieve usages for media of id " + id);
+        },
+
+        getPagedMemberReferences: function (id, options) {
+
+            var defaults = {
+                pageSize: 25,
+                pageNumber: 1
+            };
+            if (options === undefined) {
+                options = {};
+            }
+            //overwrite the defaults if there are any specified
+            angular.extend(defaults, options);
+            //now copy back to the options we will use
+            options = defaults;
 
             return umbRequestHelper.resourcePromise(
                 $http.get(
                     umbRequestHelper.getApiUrl(
                         "mediaApiBaseUrl",
-                        "GetReferences",
-                        { id: id })),
+                        "GetPagedMemberReferences",
+                        {
+                            id: id,
+                            pageNumber: options.pageNumber,
+                            pageSize: options.pageSize
+                        }
+                    )),
                 "Failed to retrieve usages for media of id " + id);
+        },
+
+        getPagedMediaReferences: function (id, options) {
+
+            var defaults = {
```

Comment:
There's a lot of repitition here, getting the defaults can just be one method that is shared.